src/main/groovy/wooga/gradle/build/unity/ios/internal/DefaultIOSBuildPluginExtension.groovy

```diff
@@ -18,15 +18,12 @@
 package wooga.gradle.build.unity.ios.internal
 
 import org.gradle.api.Action
-import org.gradle.api.credentials.PasswordCredentials
-import org.gradle.api.internal.artifacts.repositories.DefaultPasswordCredentials
-
 import wooga.gradle.build.unity.ios.IOSBuildPluginExtension
 import static org.gradle.util.ConfigureUtil.configureUsing
 
 class DefaultIOSBuildPluginExtension implements IOSBuildPluginExtension {
 
-    private final PasswordCredentials fastlaneCredentials
+    private final org.gradle.api.credentials.PasswordCredentials fastlaneCredentials
```

Comment:
Noob question I guess but why can't you import `org.gradle.api.credentials` instead of adding it everywhere?

Reply:
Ah, you also have some other PasswordCredentials down there for fastlane. Ok, I see :)

Reply:
Excuse me for being lazy. I had no muse to come up with a good alternative name of a `PasswordCredentials` class.